.github/workflows/ci.yml

```diff
@@ -30,10 +30,34 @@ jobs:
       branch: ${{ fromJSON(steps.github.outputs.result).branch }}
 
     steps:
+      - name: Get repo and branch name
```

Comment:
Move this to the top to prevent weird looking errors if the other components fail.